join_request/carlosala.yml

```
@@ -0,0 +1,39 @@
+collective: Potoc
+member:
+  name: Carlo Sala
+  address: 16JskuojL6mSp6HNcjiHYa9jqksWbLD8L9YGWU1ppiPWQ9sa
+  github: carlosala
```

please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  address: 16JskuojL6mSp6HNcjiHYa9jqksWbLD8L9YGWU1ppiPWQ9sa
  matrix:
  github: carlosala
```

---

Done, thanks!